doc/themes.md

```diff
@@ -89,10 +89,14 @@ warning_bg = { link = "good_bg" }
 [icons]
 icons = "awesome6"
 [icons.overrides]
-bat = " | | "
-bat_full = " |X| "
-bat_charging = " |^| "
-bat_discharging = " |v| "
+bat = [
```

Comment:
We could also just make this `bat = ["|E|", "|_|", "|=|", "|F|"]` like in the `examples/config.toml`. I didn't see this at first because I forgot to rebase for I started.